### ISSUE TITLE
Job Composer navigation bar is different

### ISSUE THREAD

**Issue by ashton22305** [OSC/ondemand#4954]
The navigation bar in the Job Composer is different from the rest of OOD without clear identification, a violation of[ WCAG Success Criterion 3.2.5](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-no-extreme-changes-context.html)

**Comment by johrstrom** [comment]
Note that this also applies to the shell application.

**Comment by moffatsadeghi** [comment]
After a team discussion today, we need to look into the details of the WCAG requirements and whether there are exceptions with the job composer because it is a subset of webpages in itself.

**Comment by moffatsadeghi** [comment]
Similarly with #4943, what is the plan for this issue for 4.2?

**Comment by johrstrom** [comment]
I'm closing this issue as won't do. The WCAG guideline for Consistent Navigation is for "a set of webpages". The job composer is a logical different "set of webpages" because the url is different  - `/pun/sys/myjobs` vs. `/pun/sys/dashboard`. Everything under those to paths (every set of webpages) has the same navigation.
